config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -249,6 +249,7 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-containerd, sig-node-cri
     testgrid-tab-name: containerd-node-conformance
+    testgrid-alert-email: kubernetes-sig-node-test-failures@googlegroups.com
```

Comment:
couple tests here are flaky: https://testgrid.k8s.io/sig-node-containerd#containerd-node-conformance

Will need to look into it. But I agree this tab is critical